content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/customheaderlibs.html

```diff
@@ -16,4 +16,7 @@
 
 <!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   ~ Create overlay of this file to include custom client libs in the page header
-  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
\ No newline at end of file
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+  
+<sly data-sly-test="${currentStyle.dataLayer}" data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"
+     data-sly-call="${clientlib.js @ categories='core.wcm.components.page.v2.digitalData'}"></sly>
```

Comment:
Should this be here or in the `footer.html`? These files are intentionally left empty on the existing templates to allow for customizations on individual projects. By adding this here, we now require that consumers copy this into their override of this file, else they loose the datalayer.